packages/docs/docs/docs/testing.md

```diff
@@ -3,85 +3,152 @@ id: testing
 
-## Example
+## Setting up a sample project
 
-Given a small project, with a `Sample` contract, linked to an EVM package that provides a `ERC20` token implementation:
+The following section describes a succinct way in how a simple ZeppelinOS project can be set up. If you already have a project set up, you may skip to the next section.
 
-```json
-{
-  "zosversion": "2.2",
-  "name": "my-project",
-  "version": "0.1.0",
-  "contracts": {
-    "Sample": "Sample"
-  },
-  "dependencies": {
-    "openzeppelin-eth": "2.0.2"
+_If you don't understand what's going on in this section, please refer to the Quickstart guides of the documentation, specifically the (Deploying your first project)[https://docs.zeppelinos.org/docs/deploying.html], (Upgrading your project)[https://docs.zeppelinos.org/docs/upgrading.html] and (Linking to EVM packages)[https://docs.zeppelinos.org/docs/linking.html] guides. These guides provide detailed explanations on how a basic ZeppelinOS project works._
+
+Create a new project by running:
+
+```console
+mkdir my-project
+cd my-project
+npm init --yes
+npm install zos zos-lib openzeppelin-eth truffle chai
+```
+
+Now, run:
+
+```console
+npx zos init my-project
+```
+
+Lets add a simple contract to the project, create the file `contracts/Sample.sol`:
+
+```solidity
+pragma solidity ^0.5.0;
+
+contract Sample {
+  function greet() public pure returns (string memory) {
+    return "A sample";
   }
 }
 ```
 
-To set up the full project in a test file, first import `TestHelper` from `zos`:
-```js
-import { TestHelper } from 'zos';
-```
+Now, add your contract to the ZeppelinOS project:
 
-Then invoke it in the test suite setup, optionally including a set of options to be used when deploying the contracts (such as `from`, `gas`, and `gasPrice`):
-```js
-beforeEach(async function () {
-  this.project = await TestHelper({ from: owner })
-});
+```console
+npx zos add Sample
 ```
 
-In a test, you can generate an instance for your contracts via the `createProxy` function of the project:
+And link your ZeppelinOS project to the `openzeppelin-eth` EVM package:
 
-```js
-const Sample = artifacts.require('Sample')
-it('should create a proxy', async function () {
-  const proxy = await this.project.createProxy(Sample);
-  // Use proxy ...
-})
+```
+npx zos link openzeppelin-eth
 ```
 
-The full code for the sample test file is:
+## Writing the test script
 
-```js
-import { TestHelper } from 'zos'
+To be able to use ES6 Javascript syntax in your tests, make sure you [set up babel in your project](https://docs.zeppelinos.org/docs/faq.html#how-do-i-use-es6-javascript-syntax-in-my-tests).
```

Comment:
Given that we have removed the requirement to use `import`, let's rephrase this to something like "In case you want to use ES6...". We could even move it to a separate note (using `>`).